src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -96,8 +101,14 @@ public String getNamespace() {
         return namespace;
     }
 
-    public Cloud getCloud() {
-        return Jenkins.getInstance().getCloud(getCloudName());
+    @Nonnull
+    public KubernetesCloud getCloud() {
```

Comment:
I have to admit that I don't understand enough about this, but could this be a binary compatibility issue?

Reply:
I restored the original method.